grails-app/controllers/org/pih/warehouse/UrlMappings.groovy

```diff
@@ -446,6 +446,11 @@ class UrlMappings {
             action = [GET:"currencies"]
         }
 
+        "/api/unitOfMeasure/options" {
```

Comment:
I am not sure why the previous one is in a singular version, but we should stick to plurals I think `/api/unitOfMeasures`. Perhaps we could add a second plural version and mark somehow the previous one as deprecated.